source/_docs/z-wave/adding.markdown

```diff
@@ -3,81 +3,58 @@ title: "Z-Wave Devices - Adding and Removing"
+To add a [Z-Wave device](/docs/z-wave/devices/):
 
-## Adding Non-Secure Devices
+1. Go to the [Z-Wave control panel](/docs/z-wave/control-panel/).
+2. Click **Add Node** in the *Z-Wave Network Management* card, or **Add Node Secure** for secure devices like locks. This puts your [Z-Wave controller](/docs/z-wave/controllers/) in "inclusion" mode.
+3. Activate your device by following the instructions provided with it. Usually this involves pressing a button.
+4. Make sure the device is in its final location, then click **Heal Network**. This is optional but helps optimize network speed.
 
-To add (include) a non-secure Z-Wave [device](/docs/z-wave/devices/) to your system:
-
-1. Go to the [Z-Wave control panel](/docs/z-wave/control-panel/) in the Home Assistant frontend
-2. Click the **Add Node** button in the *Z-Wave Network Management* card - this will place the controller in inclusion mode
-3. Activate your device to be included by following the instructions provided with the device
-4. With the device in its final location, run a *Heal Network*
-
-Don't use this for [secure devices](/docs/z-wave/adding/#adding-secure-devices), since this is likely to limit the features the device supports.
+When you add a device, it may initially appear without a specific entity ID (e.g., `zwave.__`) or other identifying information. *Heal Network* should speed the process of populating this information. You *might* need to restart Home Assistant for the entity ID to appear.
 
 <div class='note warning'>
-
-Don't use the OpenZWave control panel (OZWCP), **or the physical button on a controller**, to add or remove devices. Many devices will only send the information about their capabilities at the time you include them. If you use the OpenZWave control panel, or the button on a device, then Home Assistant won't have that information. Using the physical button on a controller will also result in a non-security inclusion being performed, which may limit the features the device supports.
-
+Some Z-Wave USB sticks have a physical "inclusion" button to add devices; <b>don't use it</b>. Likewise, don't add devices directly through other tools like <a href="https://github.com/OpenZWave/open-zwave-control-panel">OpenZWave control panel</a>. Many devices only send capabilities information at the time you add them, so if you add them outside of Home Assistant this information will be missing.
```

Comment:
```suggestion

Some Z-Wave USB sticks have a physical "inclusion" button to add devices; **don't use it**. Likewise, don't add devices directly through other tools like [OpenZWave control panel](https://github.com/OpenZWave/open-zwave-control-panel). Many devices only send capabilities information when you add them, so if you add them outside of Home Assistant, this information will be missing.

```

Reply:
for reasons that aren't very clear to me, this doesn't work. `bundle exec rake preview` and then previewing the page shows the markdown isn't processed:

<img width="740" alt="Screen Shot 2020-11-06 at 4 20 14 PM" src="https://user-images.githubusercontent.com/97944/98420138-224ce480-204c-11eb-8bc0-2414e4174d39.png">

Reply:
I've added new lines. Please read our developer documentation.

https://developers.home-assistant.io/docs/documenting/create-page#html

Reply:
d'oh, sorry for not reading the manual. appreciate the patience.